site/docs/v1/tech/apis/_refresh-token-response-body-base.adoc

```diff
@@ -0,0 +1,64 @@
+[.api]
+ifeval::["{base_field_name}" != "refreshToken"]
+[field]#refreshTokens# [type]#[Array]#::
+An array of Refresh Tokens
+endif::[]
+
+[field]#{base_field_name}.applicationId# [type]#[String]#::
+The Id of the Application for which this token is authorized to be exchanged for a new Access Token.
+
+[field]#{base_field_name}.device# [type]#[String]# [deprecated]#Deprecated#::
+A unique device identifier to represent the device for which this Refresh Token was issued.
++
+[deprecated]#Removed in version 1.5.0# This parameter will no longer be returned in the response body.
```

Comment:
Up to you.. but for some of these really old deprecation warnings, I'm ok with dropping them. 1.5.0 was released in March of 2019. ( https://fusionauth.io/docs/v1/tech/archive/release-notes#version-1-5-0 ).

Reply:
I think 30 versions ago is fine to drop. Will add that to the devdocs guidelines.